src/swarm-tui/Swarm/TUI/Controller.hs

```diff
@@ -1469,23 +1470,27 @@ handleInventoryListEvent ev = do
 handleInventorySearchEvent :: BrickEvent Name AppEvent -> EventM Name AppState ()
 handleInventorySearchEvent = \case
   -- Escape: stop filtering and go back to regular inventory mode
-  EscapeKey ->
+  EscapeKey -> do
+    resetViewport infoScroll
```

Comment:
I'm not sure of the desired behavior in this case. If a user searches in the inventory list, scrolls a little bit in the info panel and then presses `Esc` in the inventory list, then do we want to reset the viewport? Or should we keep the state to where the user scrolled before pressing `Esc`?

Reply:
I think in this case we should just keep the state to where the user scrolled before pressing `Esc`.